app.js

```diff
@@ -20,24 +21,23 @@ app.get("/api/articles/:article_id", getArticle);
 
 app.get("/api/articles/:article_id/comments", getArticleComments);
 
-app.post("/api/articles/:article_id/comments", postArticleComment)
+app.post("/api/articles/:article_id/comments", postArticleComment);
 
-app.use((err, req, res, next) => {
-
-  if (err.status === 404) {
-    res.status(404).send({ msg: "Article id is not found" });
-  }
-  next(err);
-});
+app.patch("/api/articles/:article_id", patchArticle),
+  app.use((err, req, res, next) => {
+    if (err.status === 404) {
+      res.status(404).send({ msg: "Article id is not found" });
+    }
```

Comment:
You shouldn't need to handle custom errors like this. If you receive a custom error send that custom error's code and message, no need to write them again.